internal/app/commands/sandbox/list.go

```diff
@@ -15,11 +15,10 @@ var List = cli.Command{
 	Usage:   "List all sandboxes",
 	Action: func(c *cli.Context) error {
 		rData := common.ReadRuntimeData()
-		myOs := util.GetCurrentOs()
 
 		for _, box := range listSandboxes("") {
 			version := parseDistroVersion(box.Distro, false)
-			boxVersion := formatSandboxListItemName(box.Name, version, myOs)
+			boxVersion := formatSandboxListItemName(box.Name, version, util.GetCurrentOsWithArch())
```

Comment:
Do we have to call `util.GetCurrentOsWithArch()` in the loop or can it be called outside ?

Reply:
@pmi Thanks, I fixed it